sdk/cpp/Dockerfile

```diff
@@ -27,6 +27,7 @@ FROM ubuntu:focal as runtime
 RUN apt-get update
 # Runtime
 RUN apt-get install -y python3
+RUN apt-get install -y python3-dev      # necessary to include libpython
```

Comment:
without this `/lib/x86_64-linux-gnu/libpython3.8.so.1.0` does not exist in the runtime container which causes the obvious problem

```
cobra_cli: error while loading shared libraries: libpython3.8.so.1.0: cannot open shared object file: No such file or directory
```